n2ws/manifest.json

```diff
@@ -26,9 +26,9 @@
   "assets": {
     "configuration": {},
     "dashboards": {
-	  "N2WSBackup&Recovery-EntitiesSpecificDashboard": "assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json",
-	  "N2WSBackup&Recovery-GraphicalVersion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
-	  "N2WSBackup&Recovery-GraphicalVersion-Areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json"
+	  "N2WSBackup&Recovery-EntityTypesDetails": "assets/dashboards/N2WSBackup&Recovery-EntityTypesDetails.json",
```

Comment:
The keys in this json cannot be changed once they have been submitted. It was already mentioned on your previous PR https://github.com/DataDog/integrations-extras/pull/760#discussion_r525258661

Reply:
I didn't touch the menifest.json, these changes probably came from git pull.
I'll revert this file.

Reply:
It looks like you have renamed all dashboard json files, is that expected? If you do so you need to rename the values in these key-value pairs.

Reply:
No, it is not expected. You gave me a comment about this issue, so I reverted the names back to what it was before.
I copied the current names from your repository on Github in order to make sure I'm not changing anything in the manifest.json in this Pull Request.

Reply:
You cannot change the keys in these pairs but if you change the filenames then you need to change the values

Reply:
I didn't change the filenames. Probably an old file from my repository accidently entered my first commit when I merged, I reverted the change.
The only thing that should be changed due to this merge request are the dashboards JSON files.